src/components/Editor/Runners/PythonRunner/PythonRunner.jsx

```diff
@@ -16,7 +16,7 @@ import {
 } from "../../../../redux/EditorSlice";
```

Comment:
The changes have fixed the issue on the Code Editor website, but it's started having the same bug on Code Editor